apps/examples/sensorbd_demo/examples/spi_k6ds3.c

```diff
@@ -163,9 +163,8 @@ void k6ds3_main(int argc, char *argv[])
 		gy = (gyl | (gyh << 8));
 		gz = (gzl | (gzh << 8));
 
-		printf("ACCEL:x(0x%04X) y(0x%04X) z(0x%04X) GYRO:x(0x%04X) y(0x%04X) z(0x%04X)\n",
-			ax, ay, az, gx, gy, gz);
+		printf("ACCEL:x(0x%04X) y(0x%04X) z(0x%04X) GYRO:x(0x%04X) y(0x%04X) z(0x%04X)\n", ax, ay, az, gx, gy, gz);
```

Comment:
Do you really think so? Looking at the coding standard document, I cannot find any mandatory clause that prohibits this. In addition, [M01][IDT_M_TAB] shows an example code snippet that inserts a new line.